src/meta/contributing.md

```diff
@@ -21,38 +21,36 @@ To begin contributing to the docs, fork the `firefox-data-docs` repo.
 ## Building the Documentation
 
 The documentation is rendered with [mdBook](https://github.com/rust-lang/mdBook).
-
-To build the documentation locally, you'll need additional preprocessors:
+We use a fork named [mdbook-dtmo][] that includes multiple different preprocessors:
 
 * [mdbook-toc](https://github.com/badboy/mdbook-toc/releases)
 * [mdbook-mermaid](https://github.com/badboy/mdbook-mermaid/releases)
 * [mdbook-open-on-gh](https://github.com/badboy/mdbook-open-on-gh/releases)
```

Comment:
Let's just take these out -- we can just keep track of this information in mdbook-dtmo and won't have to remember to update this.